sdk/Lusid.Sdk.Tests/tutorials/Ibor/Portfolios.cs

```diff
@@ -283,6 +285,16 @@ public void List_Scopes()
 
             Assert.That(scopes.Values.Count(), Is.GreaterThan(0));
         }
+        
+        [Test]
+        public void GetPortfolio_WhenPortfolioNotFound_Throws404Error()
```

Comment:
I think this should be `_ThrowsApiException`, the test for the 404 behaviour should be in LUSID itself

Reply:
fair point!